stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -232,6 +232,17 @@ macro code_lowered(ex0...)
     end
 end
 
+macro time_imports(ex)
+    quote
+        try
+            Base.TIMING_IMPORTS[Threads.threadid()] += 1
```

Comment:
With Julia having thread migration now, this kind of code is a bit fishy.
You can no longer guarantee that you receive the same value from threadid on the same task.

I would simplify this to one `Atomic` in the system. (Which would also help with esoteric code doing `using` on a different task).

Reply:
Ok, that sounds good. Thanks. Updated